keep-ui/app/(keep)/workflows/builder/editors.tsx

```
@@ -355,334 +347,377 @@
+                              color="gray"
+                              icon={FunnelIcon}
+                            >
+                              Add Filter
+                            </Button>
+                          </div>
+                          {properties.alert &&
+                            Object.keys(properties.alert as {}).map(
+                              (filter) => {
+                                return (
+                                  <>
+                                    <Subtitle className="mt-2.5">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)